lib/import-to-globals.js

```diff
@@ -118,8 +123,9 @@ function importToGlobals({ast, code, names}) {
         }
       }
       const source = getDynamicImportSource(node);
-      if (names.hasOwnProperty(source)) {
-        writeDynamicImport(code, node, names[source]);
+      const name = source && getName(source);
+      if (name) {
+        writeDynamicImport(code, node, getDynamicWrapper(name) || name);
```

Comment:
Why does it fall back to `name`? It seems that it will replace the entire `import` function with a single identifier.

Reply:
Updated